policies/compatibility.rego

```diff
@@ -331,6 +340,71 @@ deny contains back_comp_violation(description, group_id, "") if {
 }
 
 
+# Rule: Detect Removed Resources
+#
+# This rule checks for stable resources that existed in the baseline registry
+# but are no longer present in the current registry. Removing resoruces
+# is considered a backward compatibility violation.
+#
+# In other words, we do not allow the removal of a resource once added
+# to the registry. It must exist SOMEWHERE, but may be deprecated.
+deny contains back_comp_violation(description, group_id, "") if {
+    # Find data we need to enforce
+    some resource in baseline_resources
+    resource.stability == "stable"
```

Comment:
we don't remove experimental attributes or metrics - should we allow removing experimental resources?

Reply:
We haven't made guarantees on resource yet.  I think we COULD prevent removal, but I took a "make progress" stance on this PR.  At a minimum, let's protect what we stabilized.  Going forward, we need to understand what constitutes breaking for resource.